core/src/datayoga_core/resources/schemas/connections/kafka.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "Kafka",
+  "description": "Schema for configuring Kafka connection parameters",
+  "type": "object",
+  "properties": {
+    "type": {
+      "description": "Connection type",
+      "type": "string",
+      "const": "kafka"
+    },
+    "bootstrap_servers": {
+      "description": "Kafka Hosts",
```

Comment:
Is it comma separated list? Add such a note or have another one in the example (line 22).